store/sqlstore/status_store.go

```diff
@@ -99,6 +101,38 @@ func (s SqlStatusStore) GetByIds(userIds []string) ([]*model.Status, error) {
 	return statuses, nil
 }
 
+func (s SqlStatusStore) UpdateExpiredDNDStatuses() ([]*model.Status, error) {
+	queryString := fmt.Sprintf(`
+		UPDATE
+			Status
+		SET
+			DNDEndTimeUnix = %v, Manual = %v,
+			PrevStatus = '%v', Status = PrevStatus
+		WHERE
+			(Status = '%v' AND DNDEndTimeUnix <= %v AND DNDEndTimeUnix > %v)
+		RETURNING
+			UserId, Status`,
+		-1, false, model.STATUS_DND, model.STATUS_DND, time.Now().UTC().Unix(), 1,
+	)
```

Comment:
Do no use string building here, it is dangeorus and can lead to SQL injection. We prefer to use squirrel library to build the sql queries.

Reply:
I tried using it but there wasn't a way to update and get updated rows in a same query using squirrel library as there is no method to add "RETURNING" statement in query. If we want to use squirrel then need to make two db calls. one to get the statuses which meet our criteria and then update them. we need to perform both operations in a transaction.

Reply:
You can use the query builder normally and then use `.Suffix("RETURNING UserId, Status")`, for example.